yamllint/conf/relaxed.yaml

```diff
@@ -26,3 +26,5 @@ rules:
   line-length:
     level: warning
     allow-non-breakable-inline-mappings: yes
+  truthy:
+    level: warning
```

Comment:
I'd rather have it disabled in `relaxed` rule. I know a bunch of people that are really attached to their `yes`/`no` in Ansible...